src/app/components/card/card.component.ts

```diff
@@ -13,6 +13,9 @@ export class CardValue {
   avatar?: string;
   link?: string;
   menuItem?: any;
+  selfLink?: string;
```

Comment:
@sanbornsen This has been resolved here https://github.com/fabric8-ui/fabric8-planner/pull/2068/commits/0ea8061c6b989ed21feb04c520513379973b0d76